cargo-apk/README.md

```diff
@@ -45,6 +45,14 @@ assets = "path/to/assets_folder"
 # Defaults to package name.
 apk_name = "myapp"
 
+# `default` (or unspecified) - Debug symbols are not treated specially
+# `strip` - Debug symbols are stripped from the shared libraries before being
+#   copied into the APK
+# `split` - Functions the same as strip, except the debug symbols are written to
+#   the apk output directory alongside the stripped shared libraries, but with a
+#   `.dwarf` extension.
+strip = "default"
```

Comment:
I don't know what this looks like on your machine, but what's wrong with:

```suggestion
# `default` (or unspecified) - Debug symbols are not treated specially
# `strip` - Debug symbols are stripped from the shared libraries before being
#           copied into the APK
# `split` - Functions the same as strip, except the debug symbols are written to
#           the apk output directory alongside the stripped shared libraries, but
#           with a `.dwarf` extension.
strip = "default"
```

One-upping this, can we do something to align it with the `-` for `` `default` (or unspecified) - ``?